catalog/spatial/registry/registry-federation-admin-service-impl/src/test/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitializationTest.java

```diff
@@ -113,13 +122,38 @@ public void initWithNoPreviousEntry() throws Exception {
     public void initWithPreviousPrimaryEntry() throws Exception {
         testMetacard.setAttribute(new AttributeImpl(RegistryObjectMetacardType.REGISTRY_ID,
                 "registryId"));
+        testMetacard.setAttribute(new AttributeImpl(Metacard.TITLE, TEST_SITE_NAME));
         when(federationAdminService.getLocalRegistryIdentityMetacard()).thenReturn(Optional.of(
                 testMetacard));
         identityNodeInitialization.init();
         verify(federationAdminService, never()).addRegistryEntry(any(Metacard.class));
     }
 
     @Test
+    public void initWithPreviousEntryWithNameChange() throws Exception {
+        System.setProperty(SystemInfo.SITE_NAME, CHANGED_TEST_SITE_NAME);
+
```

Comment:
That's a fair concern, I think there's actually a different way I can be doing this that doesn't actually require a path at all. I'm going to try and update it now.